client/tsconfig.json

```diff
@@ -38,9 +30,8 @@
     "**/*.ts",
     "**/*.tsx",
     "src/styles/button.js",
-    ".next/types/**/*.ts"
+    ".next/types/**/*.ts",
+    "src/components/headerLogo.tsx"
```

Comment:
I never noticed this, but do we know what the `/button.js` is doing in here? I don't believe we have that as a file. Do you mind removing that line?

Reply:
I actually didn't manually add anything to this file, but it was changed after I made changes to other files. I can remove it if you know that it shouldn't be there.